packages/background/src/controllers/block-updates/BlockFetchController.ts

```diff
@@ -372,9 +373,15 @@ export class OffChainBlockFetchService {
                 () =>
                     httpClient.get<{
                         bn: string;
-                    }>(`${OFF_CHAIN_BLOCK_FETCH_SERVICE_URL}/bn`, {
-                        c: chainId,
-                    }),
+                    }>(
+                        `${OFF_CHAIN_BLOCK_FETCH_SERVICE_URL}/bn`,
+                        {
```

Comment:
I don't pretty much like the shape of this function call. The `undefined` parameters shouldn't be necessary to be specified. 

I think we should move from N Parameters to a single param:

Current signature: 

```typescript
  get<T, P extends Record<string, any> | undefined = Record<string, any>>(
        url: string,
        params?: P,
        timeout?: number,
        cache?: RequestCache
        cache?: RequestCache,
        headers?: Record<string, string>
    ): Promise<T>;
```

new proposesed signature

```typescript

type RequestOptions extends RequestInit = {
  params?: P,
  timeout?: number
}

const defaultOptions  = {
   method: 'GET',
}

 get<T, P extends Record<string, any> | undefined = Record<string, any>>(
        url: string,
        options: RequestOptions = defaultOptions
    ): Promise<T>;
```
Maybe the method is not necessary as the function itself is named `get` but for the `post` function it would be required (defaulted to POST).


What do you think? @leablock @rodr-r